CMakeLists.txt

```
@@ -141,6 +141,33 @@ pkg_check_modules(ExternalModules REQUIRED IMPORTED_TARGET ${pkg-module-spec})
 find_package(ZLIB REQUIRED)
 find_package(Threads REQUIRED)
 
+option(ENABLE_PODOFO "Compile with PoDoFo export support (Requires PoDoFo >= 1.0.0 beta 2)" OFF)
+if (ENABLE_PODOFO)
+    find_package(PoDoFo REQUIRED)
+    message(STATUS "Found PoDoFo: ${PoDoFo_FOUND}")
+endif ()
+
+option(ENABLE_MUPDF "Compile with MuPDF export support" OFF)
+if (ENABLE_MUPDF)
+    find_library(MUPDF NAMES libmupdfcpp* REQUIRED)
```

This doesn't pass in my environment (Arch Linux). Instead, the following works.
```suggestion
    find_library(MUPDF NAMES mupdfcpp REQUIRED)
```
Does `find_library` support wildcards?